backend/app/Models/Memo.php

```
@@ -0,0 +1,149 @@
+     */
+    protected function isDeadlineApproaching(): Attribute
+    {
+        return Attribute::make(get: function (): bool {
+            if ($this->deleted_at === null) {
+                return false;
+            }
+
+            return $this->deleted_at->copy()
+                ->addDays(self::DAYS_UNTIL_PERMANENT_DELETE - self::DEADLINE_WARNING_DAYS)
+                ->isPast();
+        });
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Database\Eloquent\Casts\Attribute' in method 'isDeadlineApproaching'. (undefined)

(StaticAccess)